LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2015 Gigwalk
```

Comment:
I would not copyright it Gigwalk when this is a fork of BiggerBoat / Paul Tondeur